code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -559,6 +559,12 @@
 	REMOVE_TRAIT(M, TRAIT_PACIFISM, id)
 	..()
 
+/datum/reagent/pax/peaceborg
+	name = "synthpax"
+	id = "synthpax"
```

Comment:
The standard for reagents seems to be capitalised names:
![image](https://user-images.githubusercontent.com/57483089/133864935-331ebeab-c52a-4548-8827-e211650dfc95.png)
```suggestion
/datum/reagent/pax/peaceborg
	name = "Synthpax"
	id = "synthpax"
```